ci/cloudbuild/builds/coverage.sh

```diff
@@ -53,7 +53,7 @@ io::log_h2 "Running coverage on non-integration tests."
 bazel coverage "${args[@]}" --test_tag_filters=-integration-test ...
 
 GOOGLE_CLOUD_CPP_SPANNER_SLOW_INTEGRATION_TESTS="instance"
-mapfile -t integration_args < <(integration::bazel_args)
+mapfile -t integration_args < <(integration::bazel_coverage_args)
 integration::bazel_with_emulators coverage "${args[@]}" "${integration_args[@]}"
```

Comment:
Consider leaving `integration.sh` alone and doing something like:

```
# With code coverage the `--flaky_test_attempts`
# works in unexpected ways. A flake produces an empty `coverage.dat` file, which
# breaks the build when trying to consolidate all the `coverage.dat` files.
#
# This combination of a "successful" test (because the flake is retried) and a
# failure later in the consolidation of coverage results easily leads the
# developer astray.
coverage_args=("${args[@]" "${integration_args[@]/--flaky_test_attempts=3/}")
integration::bazel_with_emulators coverage "${coverage_args[@]}" 
```